packages/nvlsm/nvlsm.spec

```diff
@@ -0,0 +1,92 @@
+%global nvlsm_major 2025
+%global nvlsm_minor 03
+%global nvlsm_patch 1
+%global nvlsm_ver %{nvlsm_major}.%{nvlsm_minor}.%{nvlsm_patch}
+%global __brp_check_rpaths %{nil}
+%if "%{?_cross_arch}" == "aarch64"
+%global fm_arch sbsa
+%else
+%global fm_arch %{_cross_arch}
+%endif
+
+Name: %{_cross_os}nvlsm
+Version: %{nvlsm_ver}
+Release: 1%{?dist}
+Epoch: 1
+Summary: NVIDIA Subnet Manager component
+License: LicenseRef-NVIDIA-AWS-EULA
+URL: http://nvidia.com
+
+# NVIDIA rpms from 0 to 199
+Source0: https://developer.download.nvidia.com/compute/cuda/repos/rhel9/x86_64/nvlsm-%{nvlsm_ver}-1.x86_64.rpm
+Source1: https://developer.download.nvidia.com/compute/cuda/repos/rhel9/sbsa/nvlsm-%{nvlsm_ver}-1.aarch64.rpm
+Source2: NVidiaEULAforAWS.pdf
+
+# NVLSM configuration files
+Source200: nvlsm.service
+Source201: nvlsm-tempfiles.conf
+Source202: nvlsm-ld.so.conf.in
+
+%description
+%{summary}.
+
+%prep
+# Extract Subnet Manager from the rpm via cpio rather than `%%setup` since the
+# correct source is architecture-dependent.
+mkdir nvlsm-linux-%{fm_arch}-%{nvlsm_ver}-archive
+rpm2cpio %{_sourcedir}/nvlsm-%{nvlsm_ver}-1.%{_cross_arch}.rpm | cpio -idmV -D nvlsm-linux-%{fm_arch}-%{nvlsm_ver}-archive
+
+%install
+pushd nvlsm-linux-%{fm_arch}-%{nvlsm_ver}-archive
+
+# Install binary
+install -d %{buildroot}%{_cross_bindir}
+install -p -m 0755 opt/nvidia/nvlsm/sbin/nvlsm %{buildroot}%{_cross_bindir}
+
+# Install shared libraries under nvidia libdir
+install -d %{buildroot}%{_cross_libdir}/nvidia/nvlsm
+install -p -m 0644 opt/nvidia/nvlsm/lib/*.so %{buildroot}%{_cross_libdir}/nvidia/nvlsm
+
+# Config file will be copied into /etc/nvidia to be updated with hardware guid
+install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/nvidia
+install -p -m 0644 usr/share/nvidia/nvlsm/nvlsm.conf %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/nvidia/nvlsm.conf
+
+# Configuration files that are referenced but do not need to be updated at runtime
+install -d %{buildroot}%{_cross_datadir}/nvidia/nvlsm
+install -p -m 0644 usr/share/nvidia/nvlsm/device_configuration.conf %{buildroot}%{_cross_datadir}/nvidia/nvlsm
+install -p -m 0644 usr/share/nvidia/nvlsm/grpc_mgr.conf %{buildroot}%{_cross_datadir}/nvidia/nvlsm
+
+popd
+
+# Install configuration files
+install -d %{buildroot}%{_cross_unitdir}
+install -p -m 0644 %{S:200} %{buildroot}%{_cross_unitdir}
+
+install -d %{buildroot}%{_cross_tmpfilesdir}
+install -p -m 0644 %{S:201} %{buildroot}%{_cross_tmpfilesdir}/nvlsm.conf
+
+install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/ld.so.conf.d/
+# We need to add `_cross_libdir` to the paths loaded by the ldconfig service
```

Comment:
nit: we should update all other comments, but we don't just add `%{_cross_libdir}`, what's added is `%{_cross_libdir}/nvidia/nvlsm`.